e2etest/bgp_test.go

```diff
@@ -1167,16 +1170,41 @@ func frrIsPairedOnPods(cs clientset.Interface, n *frrcontainer.FRR, ipFamily str
 }
 
 func createFRRContainers(c ...containerConfig) ([]*frrcontainer.FRR, error) {
+	m := sync.Mutex{}
 	frrContainers = make([]*frrcontainer.FRR, 0)
 	g := new(errgroup.Group)
 	for _, conf := range c {
 		conf := conf
 		g.Go(func() error {
+			toFind := map[string]bool{
+				"zebra":    true,
+				"watchfrr": true,
+				"bgpd":     true,
+				"bfdd":     true,
+			}
 			c, err := frrcontainer.Start(conf.name, conf.nc, conf.rc, containersNetwork, hostIPv4, hostIPv6)
 			if c != nil {
+				err = wait.PollImmediate(time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
5 minutes isn't a bit too long?

Reply:
That's the worst case scenario, I think I saw occurrences of it failing in minutes in my previous attempts.
If everything works properly, it will be faster. Note that the daemons start, soon or later, it's not a test that may fail.